src/core/pyproxy.ts

```diff
@@ -243,6 +252,16 @@ function pyproxy_new(
     target,
     is_sequence ? PyProxySequenceHandlers : PyProxyHandlers,
   );
+  if (!isAlias && !dontGCRegister) {
```

Comment:
I think we can use a positive term here: e.g. `GCRegister` than `!dontGCRegister`.